inst/css/BS4/pkgdown.sass

```diff
@@ -233,13 +198,6 @@ h6.dropdown-header {
   transform: translateY(-100%);
 }
 
-/* mark.js ----------------------------*/
```

Comment:
Where did this go?

Reply:
It's now

```css
mark {
  background: linear-gradient(-100deg,
    hsla(48,92%,75%,.3),
    hsla(48,92%,75%,.7) 95%,
    hsla(48,92%,75%,.1)
  )
}
```

and it's at line 401 (this was a duplicate).